libvast/src/system/make_pipelines.cpp

```diff
@@ -133,10 +134,12 @@ make_pipelines(pipelines_location location, const caf::settings& settings) {
     if (!name)
       return caf::make_error(ec::invalid_configuration, "pipeline name must "
                                                         "be a string");
-    auto events = caf::get_if<std::vector<std::string>>(&(*pipeline)["events"]);
-    if (!events)
-      return caf::make_error(ec::invalid_configuration,
-                             "pipeline event types must be a list of strings");
+    auto events = detail::unpack_config_list_to_vector<std::string>(
+      (*pipeline)["events"]);
+    if (!events) {
+      VAST_ERROR("Unable to extract events from pipeline config");
```

Comment:
Please stick to the usual noun-verb-object order for log messages here and start in lowercase.